devfile.yaml

```diff
@@ -1,26 +1,44 @@
 specVersion: 0.0.1
 name: chectl
 projects:
-  - name: chectl
+  - 
+    name: chectl
     source:
       type: git
       location: 'https://github.com/che-incubator/chectl.git'
 components:
-  - alias: theia-ide
+  - 
+    id: eclipse/che-theia/next
     type: cheEditor
-    id: eclipse/che-theia/1.0.0
-  - type: chePlugin
+  - 
     id: eclipse/che-machine-exec-plugin/0.0.1
+    type: chePlugin
```

Comment:
I don't think that this one is needed. I mean if you omit it it should be automatically added. So let's remove this line and make the devfile 3 lines shorter

Reply:
ok